spacewalk/uyuni-setup-reportdb/bin/uyuni-setup-reportdb

```diff
@@ -516,9 +604,18 @@ while true ; do
     shift
 done
 
+case $HOST in
+    "localhost"|$(hostname -s)|$(hostname -f))
+        EXTERNALDB=0 ;;
+    *)
+        EXTERNALDB=1 ;;
+esac
+
 case $1 in
         create) create
             ;;
+        create_remote) create_remote
+            ;;
```

Comment:
Seems this is not implemented. The code was integrated into create.

Reply:
thanks for catching. i will remove this.